container/core.py

```diff
@@ -454,7 +454,7 @@ def run_playbook(playbook, engine, service_map, ansible_options='',
             yaml.safe_dump(playbook, ofs)
         inventory_path = os.path.join(tmpdir, 'hosts')
         with open(inventory_path, 'w') as ofs:
-            for service_name, container_id in service_map.iteritems():
+            for service_name, container_id in iteritems(service_map):
```

Comment:
Is iteritems really needed? It's not like people are deploying 10000s of services in one file, so iteritems will actually be slower than just regular ol' `.items()`